server/use-console-timestamp.js

```diff
@@ -0,0 +1,21 @@
+const origLog = console.log.bind(console);
+const origWarn = console.warn.bind(console);
+const origErr = console.error.bind(console);
+
+const ts = () => {
+  return '[' + new Date().toISOString() + '] ';
+};
+
+const useConsoleTimestamp = () => {
```

Comment:
Are you concerned that this is also likely going to override `console.log` (and its friends) for our dependencies as well since we're money patching a global?

Reply:
I hate saying this out loud, but I almost wonder if we should use our own logging function rather than monkey-patching a Node global. We basically `console.log` about 7 times in the code base, so it's not a heavy lift.

Reply:
yes that's actually much safer. Will do